packages/react/src/themes/createTheme.ts

```diff
@@ -0,0 +1,6 @@
+import { ThemeInput, ThemePrepared } from './types'
+import withDebugId from '../lib/withDebugId'
+
+export const createTheme = <T = ThemeInput | ThemePrepared>(themeInput: T, debugId): T => {
```

Comment:
```suggestion
export const createTheme = <T = ThemeInput>(themeInput: T, debugId): T => {
```

Should it accept only `ThemeInput`?

Reply:
Some theme may be defined as `ThemePrepared` (that's how we are defining the Teams theme), or someone may want to use this function with a result of `mergedTheme`... As it is additional to the `ThemeInput` I don't see this as a problem